### PR TITLE
Improve evictionStrategy docs

### PR DESCRIPTION
**Pull request by dankenigsberg**
**What this PR does / why we need it**:
Improve docs by using proper capitalization of `evictionStrategy` and adding an example

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
NONE
```

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by nunnatsa — APPROVED** [review]
/approve 
/retest

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -944,9 +944,9 @@ Please correctly consider the implications of this option before setting it.
 `BlockUninstallIfWorkloadsExist` is the default behaviour.
 
 
-## Cluster level EvictionStrategy
+## Cluster level evictionStrategy
```
Comment (orenc1) [inline docs/cluster-configuration.md:947]: I think that at lease the title be a proper English title, not the actual API field (which is specified in the paragraph). I would suggest "_eviction strategy_" to align with other titles in the docs.

**Review by orenc1 — COMMENTED** [review]
Thanks, @dankenigsberg 

This is not directly in the PR, but could you please also fix `util` to `until` in line 955 ?

**Review by orenc1 — APPROVED** [review]
/approve

**Pull request merged**